src/Tasks/Microsoft.NET.Build.Tasks/ConflictResolution/NuGetUtilities.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using System;
+using System.IO;
+
+namespace Microsoft.NET.Build.Tasks.ConflictResolution
+{
+    static partial class NuGetUtilities
+    {
+        /// <summary>
+        /// Gets PackageId from sourcePath.
+        /// </summary>
+        /// <param name="item"></param>
+        /// <returns></returns>
+        public static string GetPackageIdFromSourcePath(string sourcePath)
+        {
+            string packageId, unused;
+            GetPackageParts(sourcePath, out packageId, out unused);
+            return packageId;
+        }
+
+        /// <summary>
+        /// Gets PackageId and package subpath from source path
+        /// </summary>
+        /// <param name="fullPath">full path to package file</param>
+        /// <param name="packageId">package ID</param>
+        /// <param name="packageSubPath">subpath of asset within package</param>
+        public static void GetPackageParts(string fullPath, out string packageId, out string packageSubPath)
+        {
+            packageId = null;
+            packageSubPath = null;
+            try
+            {
+                // this method is just a temporary heuristic until we get metadata added to items created by the .NETCore SDK
```

Comment:
Can we either log an issue for this, or implement it?  We ARE the .NET Core SDK.